File: aspnet-core/src/GWebsite.AbpZeroTemplate.Application.Share/Assets/Dto/AssetDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Abp.Domain.Entities;
using GWebsite.AbpZeroTemplate.Core.Models;

namespace GWebsite.AbpZeroTemplate.Application.Share.Assets.Dto
{
    /// <summary>
    /// <model cref="Asset"></model>
    /// </summary>
    public class AssetDto : Entity<int>
    {
        public string AssetName { get; set; }
        public string AssetCode { get; set; }
        public string AssetType { get; set; }
        public string AssetGroupName { get; set; }
        public string Description { get; set; }
        public string AssetStatus { get; set; }
        public float AssetValue { get; set; }
        public string Supplier { get; set; }
        public DateTime AddDate { get; set; }
        public string Note { get; set; }
        public string LinkofImage { get; set; }
    }
}

File: angular/src/app/gwebsite/asset/dto/asset-dto.ts
import { ComboboxItemDto } from '@shared/service-proxies/service-proxies';
import { DateTimeService } from '@app/shared/common/timing/date-time.service';
import { Moment } from 'moment';

export class AssetDto
{
    index?: number;
    assetName : string | undefined; 
    assetCode : string | undefined;
    assetType : string | undefined;
    assetGroupName : string | undefined;
    description : string | undefined;
    assetStatus : string | undefined;
    assetValue : number | undefined; 
    supplier : string | undefined;
    addDate : Date | Moment | string | undefined;
    note : string | undefined;
    linkofImage : string | undefined;
    id?: number;
}

export class GetAssetOutput
{
    asset: AssetDto;
    assets: ComboboxItemDto[];

}
